src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -69,7 +69,7 @@
                                     GetObjectsParameters,
                                     ResourceAccess, RequiredResourceAccess)
 
-from azure.mgmt.containerservice.models import ContainerServiceOrchestratorTypes
+from azure.mgmt.containerservice.models import ContainerServiceOrchestratorTypes, CreationData, Snapshot
```

Comment:
We'd better load model with specific api version and resource type is used in cli to achieve this. Otherwise, this is totally controlled by SDK, which might be unexpected. I will add suggestions to replace this global import to local import, but might miss something, please double check.